evap/contributor/forms.py

```diff
@@ -44,23 +44,26 @@ def clean(self):
 
         vote_start_datetime = self.cleaned_data.get('vote_start_datetime')
         vote_end_date = self.cleaned_data.get('vote_end_date')
+
+        # IMPORTANT: when removing following feature (auto-set a vote_start_datetime of the past to the future),
+        # make sure to alter the according test ("start date in the past -> form invalid?")!
+        # Location of test: evap/staff/tests/test_forms.py
+        #                   EvaluationFormTests.test_contributor_evaluation_form_date_validation(self):
+        if vote_start_datetime and vote_start_datetime < datetime.now():
+            self.cleaned_data['vote_start_datetime'] = round_datetime_up(round_to=5 * 60)
```

Comment:
I'd that we are overwriting user input here, so we at least need some kind of message on the next page telling the user about it. @janno42 opinions?